web/pdf_viewer.js

```diff
@@ -250,6 +250,14 @@ var PDFViewer = (function pdfViewer() {
           });
           self.container.dispatchEvent(event);
         };
+        pageView.textLayerOnAfterDraw =
+            function pdfViewLoadTextLayerOnAfterDraw() {
+          var event = document.createEvent('CustomEvent');
+          event.initCustomEvent('textlayerrendered', true, true, {
+            pageNumber: pageView.id
+          });
+          self.container.dispatchEvent(event);
```

Comment:
I'm not opposed to implementing this the way you describe (should be fewer lines of code).

The reason that I didn't do so, is that the `pagerendered` event is placed in pdf_viewer.js and not in pdf_page_view.js. Hence I figured that for the sake of consistency, `textlayerrendered` should be implemented similarly.